### PR TITLE
Remove never-supported WebExtensions features

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #6854, #6768

### CODE REVIEW

File: webextensions/match_patterns.json [hunk 1]
```diff
@@ -195,32 +195,6 @@
...
-            "support": {
-              "chrome": {
-                "version_added": false
-              },
-              "edge": {
-                "version_added": false
-              },
-              "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/1463440'>bug 1463440</a>"
```
Comment (saschanaz) [inline webextensions/match_patterns.json:209]: (This bug is closed as wontfix.)

File: webextensions/manifest/manifest_version.json [hunk 2]
```diff
@@ -85,36 +85,6 @@
               "deprecated": false
             }
           }
-        },
-        "v3": {
```
Comment (ddbeck) [inline webextensions/manifest/manifest_version.json:89]: It looks like this feature will survive, after rebasing.

**Review by ddbeck — CHANGES_REQUESTED** [review]
Hi @saschanaz, thanks for your patience while I got to this. I think this is ready to go, since there are no redirects needed on the content side. But it needs a rebase and I think the consequence of that will be that one of these features survives. Thanks again!

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**